src/components/experiment/ExperimentTable.tsx

```diff
@@ -0,0 +1,135 @@
+import { Typography } from '@mui/material';
+import { DateTime } from 'luxon';
+import React, { useEffect } from 'react';
+import { useQueryParams, NumberParam, DateParam } from 'use-query-params';
+
+import SuperMaterialTable, {
+  DefaultQueryParams,
+} from 'components/common/SuperMaterialTable';
+import ProposalEsiDetailsButton from 'components/questionary/questionaryComponents/ProposalEsiBasis/ProposalEsiDetailsButton';
+import { GetScheduledEventsCoreQuery } from 'generated/sdk';
+import { useScheduledEvents } from 'hooks/scheduledEvent/useScheduledEvents';
+import { tableIcons } from 'utils/materialIcons';
+import { getFullUserName } from 'utils/user';
+
+import { ExperimentUrlQueryParamsType } from './ExperimentUrlQueryParamsType';
+import ExperimentVisitsTable from './ExperimentVisitsTable';
+
+type RowType = GetScheduledEventsCoreQuery['scheduledEventsCore'][0];
+
+const columns = [
+  {
+    title: 'Proposal ID',
+    field: 'proposal.proposalId',
+  },
+  {
+    title: 'Principal investigator',
+    render: (rowData: RowType) => getFullUserName(rowData.proposal.proposer),
+  },
+  {
+    title: 'Proposal',
+    field: 'proposal.title',
+  },
+  {
+    title: 'Experiment start',
+    field: 'startsAt',
+    render: (rowData: RowType) =>
+      DateTime.fromFormat(rowData.startsAt, 'yyyy-MM-dd HH:mm:ss').toFormat(
```

Comment:
Formating here should be done using the standard formats. Something like here: https://github.com/UserOfficeProject/user-office-frontend/blob/develop/src/components/call/CallsTable.tsx#L218